zerocopy-derive/tests/enum_try_from_bytes.rs

```diff
@@ -268,3 +268,13 @@ enum GenericWithLifetimes<'a, 'b, X: 'a, Y: 'b> {
     Foo(::core::marker::PhantomData<&'a X>),
     Bar(::core::marker::PhantomData<&'b Y>),
 }
+
+#[derive(Clone, Copy, imp::TryFromBytes)]
+struct A;
+
+#[derive(imp::TryFromBytes)]
+#[repr(C)]
+enum B {
+    A(A),
+    A2 { a: A },
+}
```

Comment:
Could you add a UI test for the case where a field type is named `___ZerocopyAliasing`? We want to be sure this doesn't regress (i.e., start compiling successfully).

Reply:
So that we can unblock Fuchsia users who are depending on this, I'm going to merge this w/o extra tests, but I added the following to the release roadmap to track (note the last bullet, which links to this comment thread and to #1725):

<img width="611" alt="Screenshot 2024-09-23 at 4 24 38 PM" src="https://github.com/user-attachments/assets/26d33f19-bc2f-47e2-be58-3d6825016d39">